
package despacho.ws.servicios.expuestos;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 2.7.7.redhat-1
 * 2014-11-01T16:46:31.565-03:00
 * Generated source version: 2.7.7.redhat-1
 */

@WebFault(name = "OrdenDespachoYaExisteException", targetNamespace = "http://soap.interfaces.tpo.ia.fain.uade/")
public class OrdenDespachoYaExisteException_Exception extends java.lang.Exception {
    
    private despacho.ws.servicios.expuestos.OrdenDespachoYaExisteException ordenDespachoYaExisteException;

    public OrdenDespachoYaExisteException_Exception() {
        super();
    }
    
    public OrdenDespachoYaExisteException_Exception(String message) {
        super(message);
    }
    
    public OrdenDespachoYaExisteException_Exception(String message, Throwable cause) {
        super(message, cause);
    }

    public OrdenDespachoYaExisteException_Exception(String message, despacho.ws.servicios.expuestos.OrdenDespachoYaExisteException ordenDespachoYaExisteException) {
        super(message);
        this.ordenDespachoYaExisteException = ordenDespachoYaExisteException;
    }

    public OrdenDespachoYaExisteException_Exception(String message, despacho.ws.servicios.expuestos.OrdenDespachoYaExisteException ordenDespachoYaExisteException, Throwable cause) {
        super(message, cause);
        this.ordenDespachoYaExisteException = ordenDespachoYaExisteException;
    }

    public despacho.ws.servicios.expuestos.OrdenDespachoYaExisteException getFaultInfo() {
        return this.ordenDespachoYaExisteException;
    }
}
